README.md

```diff
@@ -365,6 +365,9 @@ Tests and test coverage
 Included is a set of comprehensive tests, which you can run by:
 ``./manage.py test``
 
+If you want to run selenium tests don't forget to add ``RUN_SELENIUM_TESTS = True``
+in your ``airmozilla/settings/local.py`` file also you need to install [phantomjs](http://phantomjs.org/).
+
```

Comment:
It's Markdown. Not reStructuredText. So you can just use a single back-tick. 
Also, try to keep the sentences shorter than 80 characters like the rest of the file. It's not fatal if it spills over 80 chars but that should only really be for certain code snippets that are hard to split onto multiple lines. 

Also, it's called `PhantomJS` not `phantomjs`.